mobile/src/main/java/org/openhab/habdroid/ui/WidgetListFragment.kt

```diff
@@ -87,10 +87,8 @@ class WidgetListFragment : Fragment(), WidgetAdapter.ItemClickListener {
     private val suggestedCommandsFactory by lazy {
         SuggestedCommandsFactory(requireContext(), false)
     }
-    private val dataSaverChangeListener: BroadcastReceiver? = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
+    private val dataSaverChangeListener by lazy {
```

Comment:
Does the 'by lazy' get us anything here, given it'll be constructed unconditionally in onStart? I think the old variant actually was better, just the register call could've been made dependent on null status of the listener.

Reply:
I moved the unregistration to `onStop()` in #1905. The battery saver is available for all api levels the app supports, so no need to check for it in that PR.